WcaOnRails/app/webpacker/components/Competitions/MyCompetitions/index.jsx

```diff
@@ -0,0 +1,310 @@
+                  </a>
+                  )}
+                </Table.Cell>
+                <Table.Cell>
+                  { (permissions.can_organize_competitions.scope === '*' || permissions.can_organize_competitions.scope.includes(competition.id)) && (
+                  <a href={competitionRegistrationsUrl(competition.id)}>
+                    { I18n.t('competitions.my_competitions_table.registrations') }
+                  </a>
+                  )}
+                </Table.Cell>
+                <ReportColumn competitionId={competition.id} permissions={permissions} />
+              </Table.Row>
+)}
+          />
+        ))}
+      </TableBody>
+    </Table>
+  );
+}
+
+function PastCompetitionsTable({ competitions, permissions }) {
+  return (
+    <Table striped>
+      <TableHeader>
+        <Table.Row>
+          <Table.HeaderCell>
+            {I18n.t('competitions.adjacent_competitions.name')}
+          </Table.HeaderCell>
+          <Table.HeaderCell>
+            {I18n.t('competitions.adjacent_competitions.location')}
+          </Table.HeaderCell>
+          <Table.HeaderCell>
+            {I18n.t('competitions.adjacent_competitions.date')}
+          </Table.HeaderCell>
+          <Table.HeaderCell />
+          <Table.HeaderCell />
+          <Table.HeaderCell />
+        </Table.Row>
+
+      </TableHeader>
+      <TableBody>
+        {competitions.map((competition) => (
+          <Table.Row key={competition.id}>
+            <Table.Cell>
+              <a href={competition.url}>{competition.name}</a>
+            </Table.Cell>
+            <Table.Cell>
+              {cityAndCountry(competition)}
+            </Table.Cell>
+            <Table.Cell>
+              {dateRange(competition.start_date, competition.end_date)}
+            </Table.Cell>
+            <Table.Cell>
+              {!competition['results_posted?'] && (
+                <Icon name="calendar check" />
+              )}
+            </Table.Cell>
+            <Table.Cell>
+              {competition['results_posted?'] && (
+              <Popup
+                content={I18n.t('competitions.my_competitions_table.results_up')}
+                trigger={(
+                  <Icon name="check circle" />
+                )}
+              />
+              )}
+            </Table.Cell>
+            <ReportColumn competitionId={competition.id} permissions={permissions} />
+          </Table.Row>
+        ))}
+      </TableBody>
+    </Table>
+  );
+}
+
+export default function MyCompetitions() {
+  const [isAccordionOpen, setIsAccordionOpen] = React.useState(false);
+  const [shouldShowRegistrationStatus, setShouldShowRegistrationStatus] = React.useState(false);
+
+  const { data: competitions, loading: competitionsLoading } = useLoadedData(myCompetitionsAPIUrl);
+  const { data: me, loading: meLoading } = useLoadedData(meAPIUrl);
+  const { data: permissions, loading: permissionsLoading } = useLoadedData(permissionsAPIUrl);
```

Comment:
Do we want/need any error handling here?
There's one instance where you use `competitions?.` but the rest are all `competitions.` which will crash.